coderedcms/views.py

```diff
@@ -75,18 +72,23 @@ def search(request):
                 try:
                     model = ContentType.objects.get(model=search_model).model_class()
                     results = model.objects.live().search(search_query)
-                except:
+                except search_model.DoesNotExist:  # Possible search also causes an exception, nothing found online  # noqa
```

Comment:
Put your "possible search also..." comment above or below this line, then noqa is not needed.